File: ast/statement/expression.go
package statement

import (
	"github.com/panda-io/micro-panda/ast/ast"
)

type ExpressionStatement struct {
	StatementBase
	Expression ast.Expression
}

func (e *ExpressionStatement) Validate(c ast.Context) {
	e.Expression.Validate(c, nil)
}

File: app/src/main/java/com/github/panda_io/micro_panda/ast/statement/Expression.java
package com.github.panda_io.micro_panda.ast.statement;

import com.github.panda_io.micro_panda.ast.Context;

public class Expression extends Statement {
    public com.github.panda_io.micro_panda.ast.expression.Expression expression;

    public void validate(Context context) {
        this.expression.validate(context, null);
    }
}